src/reactviews/pages/QueryResult/table/table.ts

```diff
@@ -213,10 +220,11 @@ export class Table<T extends Slick.SlickData> implements IThemable {
         this.columns.forEach((column) => {
             if (column.field) {
                 const filters =
-                    this.queryResultState.state.filterState[column.field];
+                    this.queryResultState.state.filterState[this.gridId];
                 if (filters) {
                     (<FilterableColumn<T>>column).filterValues =
-                        filters.filterValues;
+                        //@ts-ignore
```

Comment:
what didn't it like?

Reply:
It complained that I was using a string to index my array